src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.colors.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
```

Comment:
This methoid should return color, in our program color is enum, so please return propriate value here